packages/core/src/components/Sheet/index.tsx

```diff
@@ -207,13 +230,15 @@ class BaseSheet extends React.Component<Props & PrivateProps & WithStylesProps,
                 gap && animating && visible && styles.sheet_slide_in,
               )}
             >
-              <div {...css(styles.closeButton, gap && styles.closeButton_rightAlign)}>
-                <IconButton onClick={this.handleClose}>
-                  <IconClose accessibilityLabel={closeText} size={3 * theme!.unit} />
-                </IconButton>
+              <div {...css(headerShadow && styles.headerShadow)}>
+                <Spacing all={compact ? 1 : 4} bottom={0}>
+                  <Row middleAlign before={!gap && closeIcon} after={gap && closeIcon}>
```

Comment:
I'm not a fan of the close icon switching sides.

Reply:
This is what it did before, I just moved the logic into `<Row` to accommodate the header bar

Reply:
Bugged me then too :P Maybe we can standardize this somehow.

Reply:
it was part of the sheet + gap design - would be weird to have the x on the left with the gap, imo. no gap and on the left is consistent with modal.

Reply:
Can we shelve this for a different discussion / PR?